vspace/src/main/webapp/WEB-INF/views/staff/modules/slides/slide.html

```diff
@@ -143,33 +152,104 @@
             function onMouseLeave(e) {
                 $(this).removeClass("hova");
             }
+
+            function getSelectedSpace(blockId) {
+                $.ajax({
+                    url: "[(@{'/staff/module/'+${module.id}+'/slide/'+${slide.id}+'/space/'})]" + blockId,
+                    type: 'GET',
+                    success: function(result) {
+                    	if (result != null && result.space != null && result.space.name != null){
```

Comment:
I don't think `result.space.name != null` is needed here? If the space name is null, it shouldn't prevent the space block from being created. However, if there is no title, there should be some dummy text (in the space selection list as well as when displaying the block/link), e.g. "<space has to title>"